gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -209,7 +209,7 @@ def queue_policy(queue_arn, topic_arn)
   "Id": "#{queue_arn}/SQSDefaultPolicy",
   "Statement": [
     {
-      "Sid": "#{Digest::MD5.hexdigest(queue_arn)}",
+      "Sid": "#{Rails.application.config.digest_class.hexdigest(queue_arn)}",
```

Comment:
Oh in that case, yeah, we can replace with `SecureRandom.hex`, and just be done with it...no need to even configure.